Update BLAST settings styles

azangru opened the pull request:
## Description
- Updated the width of select elements in the BLAST settings section from 70px to 90px
- Since something like BLOSUM62 won't fit in 90px, set the width of the matrix selector (as well as the select elements that can appear directly above it) to 110px
- Removed the set 140px width for labels, thus bringing the select elements closer together (as in design; consulted with Andrea)
- Updated other styles to fix behaviour at breakpoints

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1875

## Deployment URL(s)
http://blast-select-width.review.ensembl.org

## Views affected
BLAST form (specifically, the BLAST parameters section)

jyothishnt commented on src/content/app/tools/blast/components/blast-settings/BlastSettings.scss:
```diff
@@ -54,8 +56,8 @@
   padding-top: 20px;
```
Update `gap: 20px`?

azangru replied:
Not sure I understand what you mean here.

jyothishnt replied:
gap as per XD is 20px. Its on line 55.

azangru replied:
I've updated the gap.

There is one thing in which the result deviates from XD: the select box for gap penalties isn't perfectly aligned with the select boxes above it on a narrow screen:

https://user-images.githubusercontent.com/6834224/234836350-366d15f6-b7ea-4d21-b3fe-60771f6e5c65.mov

There isn't a clean solution. The best I could come up with was to set a certain width (190px) on the div containing the the gap penalties; but it is fragile. I asked Andrea if we can ignore this problem, and she agreed.

jyothishnt commented:
Loosing the alignment at 1049px. May be not for this PR, but if it is an easy fix, then why not.
![Screenshot 2023-04-26 at 11 22 15](https://user-images.githubusercontent.com/6347854/234547169-4111f948-3ead-4f53-b2e1-8eaf24864e99.png)

azangru commented:
> if it is an easy fix, then why not

Updated. Removed the margin-left

There is a deeper problem, which is that at our smallest allowed width, the Submission name goes on a new line at all, when there is seemingly plenty of available space to the right; but that would be difficult to fix; so I am not touching that part.

https://user-images.githubusercontent.com/6834224/234708726-6f2d4515-66e7-401f-9b64-cdd5d65103da.mov
